app/controllers/api/rooms_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::RoomsController < Api::BaseController
+
+  skip_before_filter :authenticate_user, only: [:create]
```

Comment:
We don't want to skip this filter, since `create` should still authenticate a user (probably a copy paste error just remove this line)!